frontend/src/components/DiscordIcon.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export default function DiscordIcon({ size }) {
+  return (
+    <svg
+      viewBox="0 -28.5 256 256"
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+      preserveAspectRatio="xMidYMid"
+      fill="#currentColor"
+      height={size}
+      width={size}
+    >
+      <g id="SVGRepo_bgCarrier" stroke-width="0"></g>
+      <g id="SVGRepo_tracerCarrier" stroke-linecap="round" stroke-linejoin="round"></g>
+      <g id="SVGRepo_iconCarrier">
```

Comment:
<img width="515" height="382" alt="image" src="https://github.com/user-attachments/assets/fdc4303f-bc1c-4ab3-b09d-5f80d425abb9" />

I'm getting this error in console, using Google Chrome.
```suggestion
      <g id="SVGRepo_bgCarrier" strokeWidth="0"></g>
      <g id="SVGRepo_tracerCarrier" strokeLinecap="round" strokeLinejoin="round"></g>
      <g id="SVGRepo_iconCarrier">
```